Formula/vrpn.rb

```diff
@@ -22,18 +22,10 @@ class Vrpn < Formula
   depends_on "doxygen" => :build if build.with? "docs"
   depends_on "libusb" # for HID support
 
-  # Upstream commit fixes Xcode 9 build failure, remove in next version
-  # https://github.com/vrpn/vrpn/issues/187
-  patch do
-    url "https://github.com/vrpn/vrpn/commit/855af40d.patch?full_index=1"
-    sha256 "601335e211cee722814650a6a31cfc6a4278ae52d5e9527a2799df3dd47758b9"
-  end
-
   def install
-    ENV.libstdcxx
-
     args = std_cmake_args
     args << "-DCMAKE_OSX_SYSROOT=#{MacOS.sdk_path}"
+    args << "-DVRPN_BUILD_JAVA:BOOL=OFF"
```

Comment:
Since we have JDK installed on our CI nodes, it will otherwise attempt to automatically pick it up. See https://github.com/vrpn/vrpn/blob/version_07.34/CMakeLists.txt#L353-L372.